package.json

```diff
@@ -23,6 +25,14 @@
     "path": "^0.12.7"
   },
   "devDependencies": {
-    "nodemon": "^2.0.3"
+    "@babel/core": "^7.9.6",
+    "@babel/preset-env": "^7.9.6",
+    "@babel/preset-react": "^7.9.4",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
```

Comment:
Might want to remove duplicates. Not sure how big of a deal that is. Possibly not at all.